ui/packages/ui/src/app/pages/connectors/ConnectorTask.tsx

```diff
@@ -31,12 +32,23 @@ export const ConnectorTask: React.FunctionComponent<IConnectorTaskProps> = (
       break;
   }
 
+  let errors = [];
+  if (props.errors) {
+    errors =  props.errors;
+  }
+
   return (
     <>
       <Tooltip
         content={
           <div>
             Task {props.taskId}: {props.task}
+            <div class="errors">
```

Comment:
```suggestion
            <div className="errors">
```

Reply:
Was not able to test this. How can I have the tooltip open AND examine it in dev toolbar? pseudo-classes like hover don't work. Isn't that bad UX / accessibilty? the tooltip should always be there for disabled people and only be shown on :hover ? WDYT? should we log an issue for that? @uidoyen  @indraraj

Reply:
I logged #192  for it